code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6302,16 +6302,23 @@
 	dupeable = FALSE
 
 /datum/reagent/ethanol/scientists_serendipity/handle_special_behavior(var/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/D)
-	if(volume<10)
+	if(volume < 10)
 		glass_icon_state = "scientists_surprise"
 		glass_name = "\improper Scientist's Surprise"
 		glass_desc = "There is as yet insufficient data for a meaningful answer."
-	else
+
+	else if(volume < 50)
 		glass_icon_state = "scientists_serendipity"
 		glass_name = "\improper Scientist's Serendipity"
 		glass_desc = "Knock back a cold glass of R&D."
 		D.origin_tech = "materials=7;engineering=3;plasmatech=2;powerstorage=4;bluespace=6;combat=3;magnets=6;programming=3"
 
+	else
+		glass_icon_state = "scientists_serendipity"
+		glass_name = "\improper Scientist's Lucky Datadisk"
```

Comment:
This name doesn't match the naming convention of alliteration
> Doctor's Delight
> Scientist's Surprise
> Scientist's Serendipity

Reply:
Scientist's Sapience it is now.